custom/icds/forms.py

```diff
@@ -229,4 +229,9 @@ def clean_end_date(self):
             raise forms.ValidationError(_("Invalid date"))
         if start_date > end_date:
             raise forms.ValidationError(_("Start date cannot be greater than end date"))
+        if (end_date - start_date).days > 95:
```

Comment:
(non blocking)
that would be 3 months. A normal one month report takes like 10 hours to get generated. Allowing a period of 3 months is going to take more than a day.
Should we allow just one month i.e 31 days? If user needs for 3 months they can do three requests?
which would be done in parallel so they would get all of them in roughly 10 hours.
(feel free to punt this since this can be revisited later or take it up with app team for inputs as you feel okay)

nit: would be nice to have 95 extracted out as a constant.

Reply:
I had a conversation with @akshita-sh, going with a limit of 35.